src/bindings/utils_bindings.cpp

```diff
@@ -169,4 +169,11 @@ void bind_utils(pybind11::module_ &m) {
              &Utils::create_rolling_window_wrapper)
         .def("get_name", &Utils::get_name)
         .def("get_upper_triu_cov_wrapper", &Utils::get_upper_triu_cov_wrapper);
-}
\ No newline at end of file
+}
+
+void bind_seed_manager(pybind11::module_ &m) {
```

Comment:
I dont think we need to bind the `SeedManager` class. See my general comment in PR